python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -1016,3 +1036,52 @@ def _evaluate_auto_materialize_tick(
             )
 
         self._logger.info(f"Finished auto-materialization tick{print_group_name}")
+
+
+def invoke_sensor_for_evaluation(
```

Comment:
seems like whether to do this new thing in the asset daemon or the sensor daemon is a big fork in the road, given that we are now explicitly adding functionality and grpc calls that can already be found in the sensor daemon

Reply:
I think it is important for the short term to do this in the `AssetDaemon`. While I do think the `AssetDaemon` should be folded into the `Sensor` I think that should be done as a separate project that also pushes on horizontal scalability and other known issues.